src/for-redux-swr/SpreadoSetupImpl.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint react-hooks/rules-of-hooks: "off" */
+import {Store} from 'redux';
+import {SpreadoSetup} from '..';
+import {SpreadoSetupOptions} from '../core';
+import {getSpreadIn, setSpreadOut, useSpreadIn, useSpreadOut} from '../redux';
+
+export const EMPTY_BODY_MSG = 'In development';
+
+interface Options extends SpreadoSetupOptions {
+  store: Store;
+}
+
+export class SpreadoSetupBetweenReduxSwr implements SpreadoSetup {
```

Comment:
Would you inherit `SpreadoSetupBase` just like the way in `for-redux-react-query/SpreadoSetupImpl.ts`?